litellm/integrations/clickhouse.py

```diff
@@ -275,6 +275,7 @@ def log_event(
                 response_obj=response_obj,
                 start_time=start_time,
                 end_time=end_time,
+                end_user_id=None,
```

Comment:
No... Here is the function definition for `get_logging_payload` at `litellm.proxy.utils` 

```
def get_logging_payload(
    kwargs, response_obj, start_time, end_time, end_user_id: Optional[str]
):
    from litellm.proxy._types import LiteLLM_SpendLogs
    ....
```

As I was reading through the code, I found the required param is not passed. Since its not passed, I hope setting value as `None` is what expected. But we do have a param called `user_id` from the parent function.

@krrishdholakia Should `None` replaced with `user_id` ?